sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
 * Added Azure Container Apps resource detector.
   ([#41803](https://github.com/Azure/azure-sdk-for-net/pull/41803))
 
+* Added `Azure.Monitor.OpenTelemetry.LiveMetrics`, enabling the sending of live
```

Comment:
Link to this would be helpful https://learn.microsoft.com/en-us/azure/azure-monitor/app/live-stream?tabs=dotnet6 or https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md for users unfamiliar with live metrics.